src/css/skins/seven.less

```diff
@@ -1,301 +1,67 @@
+    .horizontally-centered-rail-element(@rail-height, @thumb-size * 1.3);
+  }
 
-    .jw-buffer {
-        background-color: #666F82;
-    }
+  .jw-volume-tip {
+    padding: 0.5em;
+  }
 
-    .jw-progress {
-        background: @active-color;
-    }
+  .jw-playlist-container {
+    left: -38%;
+  }
 
-    .jw-knob {
-        width: .6em;
-        height: .6em;
-        background-color: #fff;
-        box-shadow: 0px 0px 0px 1px rgba(0,0,0,1);
-        border-radius: 1em;
-    }
-
-    /* Timeline and horizontal volume slider */
-    .jw-slider-horizontal {
-
-        .jw-slider-container {
-            height: 0.95em;
-        }
-
-        .jw-rail,
-        .jw-buffer,
-        .jw-progress {
-            height: .2em;
-            border-radius: 0;
-        }
-
-        .jw-knob {
-            .vertically-centered-rail-element(.2em, .6em);
-        }
-
-        .jw-cue {
-            .vertically-centered-rail-element(.2em, .3em);
-
-            width: .3em;
-            height: .3em;
-            background-color: #fff;
-            border-radius: 50%;
-        }
-    }
-    
-    /* Volume slider */
-    .jw-slider-vertical {
-        .jw-rail,
-        .jw-buffer,
-        .jw-progress {
-            width: .2em;
-        }
-        .jw-knob {
-            margin-bottom: -0.3em;
-        }
-    }
-
-    /* Volume slider container */
-    .jw-volume-tip {
-        width: 100%;
-        left: -45%;
-        padding-bottom: .7em;
-    }
-
-    /* Color of video's duration time */
-    .jw-text-duration {
-        color: #666F82;
-    }
-    
-    /* Icon borders on the right side of the controlbar */
-    .jw-controlbar-right-group {
-        .jw-icon-tooltip:before,
-        .jw-icon-inline:before {
-            border-left: 1px solid #666;
-        }
-
-        .jw-icon-inline:first-child:before {
-            border: none;
-        }
-    }
-
-    /* Icons, text and container styles for the dock, in the upper right corner */
-    .jw-dock {
-        .jw-dock-button {
-            border-radius: 50%;
-            border: 1px solid #333;
-        }
-        .jw-overlay {
-            border-radius: 2.5em;
-        }
-    }
-    
-    /* Styles for active list items */
-    .jw-icon-tooltip {
-        .jw-active-option {
-            background-color: @active-color;
-            color: #fff;
-        }
-    }
-    
-    /* A min-width on the volume icon to keep the width from collapsing on toggle */
-    .jw-icon-volume {
-        min-width: 2.6em;
-    }
-    
-    /* For the overlay containers */
-    .jw-time-tip,
-    .jw-menu,
-    .jw-volume-tip,
-    .jw-skip {
-        border: 1px solid #333;
-    }
-    
-    /* Position and padding of slider thumbnail */
-    .jw-time-tip {
-        padding: 0.2em;
-        bottom: 1.3em;
-    }
-
-    /* Position of remaining overlay containers */
-    .jw-menu,
-    .jw-volume-tip {
-        bottom: .24em;
-    }
-    
-    /* Skip icon and text styles */
-    .jw-skip {
```

Comment:
Looks like you removed the skip button customizations.

As a general rule. I expect that if you take something away from seven, you add it somewhere else, but I don't see that for skip.